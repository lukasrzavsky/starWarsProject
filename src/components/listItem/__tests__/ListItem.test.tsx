import React from "react";
import renderer from "react-test-renderer";
import { ListItem } from "..";

describe("<ListItem />", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<ListItem name="Name" detail="Detail" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("has 2 children", () => {
    const tree = renderer
      .create(<ListItem name="Name" detail="Detail" />)
      .toJSON() as renderer.ReactTestRendererJSON;
    expect(tree.children.length).toBe(2);
  });
});
