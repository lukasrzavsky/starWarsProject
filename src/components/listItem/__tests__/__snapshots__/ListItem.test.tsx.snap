// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItem /> renders correctly 1`] = `
<View
  style={
    {
      "padding": 16,
    }
  }
>
  <Text
    style={
      {
        "fontSize": 18,
      }
    }
  >
    Name
  </Text>
  <Text
    style={
      {
        "fontSize": 14,
      }
    }
  >
    Detail
  </Text>
</View>
`;
